src/applications/mhv/medications/tests/util/txtConfigs.unit.spec.jsx

```diff
@@ -63,7 +63,7 @@ describe('Allergies List Config', () => {
   it('should show try again message when allergies is falsy', () => {
     const txt = buildAllergiesTXT(null);
     const msg =
-      'We couldn’t access your allergy records when you downloaded this list. We’re sorry. There was a problem with our system. Try again later. If it still doesn’t work, email us at vamhvfeedback@va.gov.';
+      'We couldn’t access your allergy records when you downloaded this list. We’re sorry. There was a problem with our system. Try again later. If it still doesn’t work, call us at 877-327-0022 (TTY: 711). We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.';
```

Comment:
hey for the phone number are you able to use the `<va-telephone />` component instead of the string literal for the phone number?

Reply:
i meant to comment this on the `src/applications/mhv/medications/util/txtConfigs.js` file not this one. 😅